packages/cli/src/commands/init.ts

```diff
@@ -36,8 +36,8 @@ export default class Init extends Command {
       throw new Error(`Directory ${project.name} exists, try another project name`);
     }
     project.repository = await cli.prompt('Git repository', {required: false});
-    project.endpoint = await cli.prompt('RPC endpoint', {
-      default: 'wss://polkadot.api.onfinality.io/public-ws',
+    project.network = await cli.prompt('Netork genesis hash', {
+      default: '0x91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3',
```

Comment:
It might be worth adding more human readable options here. Like a list of network names and the ability to add a custom genesis

Reply:
We can ask user for an endpoint and retrieve genesishash from that endpoint and ask confirm.